### ISSUE TITLE
Infinite loading

### ISSUE THREAD

**Issue by Wolfen113** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3302]
**OS:** Arch Linux, Windows 10

**GPMDP Version:**  4.6.1

**Issue Descriptions:** I left for a few days, returned, started the computer, launched the player. And it does not load, spin the load wheel, the interface does not react to any actions. This is for both systems.

**Steps to Reproduce:** I started the player after several days of absence.

**Comment by Wolfen113** [comment]
This is what the Linux terminal shows and I do not understand what it means.

*** WARNING *** The program 'Google Play Music Desktop Player' uses the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/blog/projects/avahi-compat.html>
*** WARNING *** The program 'Google Play Music Desktop Player' called 'DNSServiceRegister()' which is not supported (or only supported partially) in the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/blog/projects/avahi-compat.html>
*** WARNING *** The program 'Google Play Music Desktop Player' uses the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/blog/projects/avahi-compat.html>
*** WARNING *** The program 'Google Play Music Desktop Player' called 'DNSServiceRegister()' which is not supported (or only supported partially) in the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/blog/projects/avahi-compat.html>
*** WARNING *** The program 'Google Play Music Desktop Player' uses the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/blog/projects/avahi-compat.html>
*** WARNING *** The program 'Google Play Music Desktop Player' called 'DNSServiceRegister()' which is not supported (or only supported partially) in the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/blog/projects/avahi-compat.html>

**Comment by jostrander** [comment]
With two systems on two OS's not working I have to wonder if its something external. Did anything else change? Can you access the site via music.google.com?

**Comment by Wolfen113** [comment]
Yes, through the site the service is working, I have been listening to Google for 2 days now, because the GPMDP does not work.
I do not know what could change in the few days that I was gone that the player stopped working. And no updates nebylo. Except that....
Not long before departure, I changed the security settings of the Google profile by setting up 2-step verification. Then I disabled this setting. Could Google somehow block access to a third-party application of the GPMDP?
![image](https://user-images.githubusercontent.com/29464780/43443906-39e04cb2-949a-11e8-835a-d1919982484a.png)

**Comment by Wolfen113** [comment]
@jostrander 
It seems the problem is not with the GPTDP but with my Internet. I just found out that the music download manager on Google Play Music also does not work. He says that there is no internet connection, although in fact it is there and everything works.
  Maybe there are not enough ports to open, but I do not know what.
Could not connect to Google Play. Check your Internet connection and try again
![image](https://user-images.githubusercontent.com/29464780/43445761-118deaf8-949f-11e8-9edf-b71c33b5e360.png)

**Comment by dzek69** [comment]
No ports are needed to be open.

I could have few solutions, but all of them seems to be non-sense because:
- the issue is the same on different system
- and at the same time - it works in your browser

So this is not a system problem or connection problem. So we're like out of possible sources of the problem.

This is what i started to write, before realising that it works in your browser, you may try it anyway:

Try restarting the router. If you have ad-block on your router - disable it.

If you are located in Russia - it could be a side effect of Russian goverment fighting with Telegram: https://www.theguardian.com/world/2018/apr/17/russia-blocks-millions-of-ip-addresses-in-battle-against-telegram-app - you could use VPN or Proxy (there are dozen of "free" ones, but free vpns are often a malware/privacy issue and VPNs are illegal in Russia IIRC) to work around this issue.

If you're Ukrainian or other country that is using cyrillic alphabet and VPNs are legal there - just try any VPN.

You may also try to switch the ISP. Fastest way - just let your computer to connect to the internet through your mobile phone connection, not home connection.

**Comment by Wolfen113** [comment]
Unfortunately I do not understand what you mean by the router. I have an ADSL modem and I do not know how to set it up. The phone is not on the Internet, I can not check it. I live in Belarus, there is no telegram blocking. Just in case, I already tried to use VPN did not help.

And yet I think that the problem is in the player. If the system delete all the config files associated with the player, then the settings will be reset. After the reset and re-registration, the player partially works. Allowing to play some playlists without going to them. If I try to go to the prelist and other places the player hangs up again. If I exit the player and go back again, it also does not work, Until I delete the config files. 
![image](https://user-images.githubusercontent.com/29464780/43730409-e6d3a324-99b3-11e8-8c94-6dbbad5ffb16.png)
![image](https://user-images.githubusercontent.com/29464780/43730425-f81ba15e-99b3-11e8-940d-d64d4901cac4.png)

**Comment by Wolfen113** [comment]
I determined the method of experiments. The GPTDP does not go to endless loading if it always remains the main page, when it is turned on and off.
  Playlists can be partially played. through the button in the image above, but not all the playlists. With each new download, random playback of playlists is blocked, displaying an infinite download on the button
I hope for the next update the player will fix

**Comment by charlwillia6** [comment]
I seem to have less issues too if I do not use a custom theme.

**Comment by Wolfen113** [comment]
Earlier I wrote that partially solved the problem, it was on Windows. On Linux, this also worked for the first time. But then after another deletion of the config files the player stopped working at all. Even returning files or completely reinstalling the player did not help. 
The console issues an error.
Segmentation error (the memory stack is flushed to disk)

**Comment by BrianAllred** [comment]
That's different, check #3312.
